internal/monitors/collectd/df/metadata.yaml

```diff
@@ -20,37 +25,46 @@ monitors:
     included: false
     name: df_inodes.free
     type: gauge
+    group: ReportInodes
   - description: Number of inodes reserved for the super user.
     included: false
     name: df_inodes.reserved
     type: gauge
+    group: ReportInodes
   - description: Number of inodes that are used.
     included: false
     name: df_inodes.used
     type: gauge
+    group: ReportInodes
   - description: Free disk space on the file system, expressed as a percentage.
     included: false
     name: percent_bytes.free
     type: gauge
+    group: ValuesPercentage
   - description: Reserved disk space on the filesystem, expressed as a percentage.
     included: false
     name: percent_bytes.reserved
     type: gauge
+    group: ValuesPercentage
   - description: Used disk space on the file system, expressed as a percentage.
     included: false
     name: percent_bytes.used
     type: gauge
+    group: ValuesPercentage
   - description: Free inodes on the file system, expressed as a percentage.
     included: false
     name: percent_inodes.free
     type: gauge
+    group: ValuesPercentage
   - description: Reserved inodes on the file system, expressed as a percentage.
     included: false
     name: percent_inodes.reserved
     type: gauge
+    group: ValuesPercentage
```

Comment:
Ben and I discussed this before and I think we decided that for metrics thats appear to fall into two groups we would choose the primary one. In this case it seems like inodes is the primary one.